### DISCUSSION TITLE
Usable maps in newly reconstructed parent grains

### DISCUSSION THREAD

**Discussion by lebraunjames** [mtex-toolbox/mtex#1851] [Ask Anything] [answered]
After conducting parent reconstruction, are the following maps reliable for representing the properties of the parent microstructure:
-parent KAM map
-parent LAGBs and CSL boundaries
-parent Phase map? 

Or should I exclusively use the IPF map and the HAGBs of the reconstructed parent grains for this purpose?

Thank you in advance for your insights.

**Comment by nyyssont** [comment, marked as answer]
I would say no to the first two points. You will get CSL 3 boundaries, but they will not be accurately placed due to inherent difficulties related to their reconstruction - see https://scripts.iucr.org/cgi-bin/paper?S1600576721011560 for details.

The last point I don't quite understand - the algorithms attempt to reconstruct the entire given child phase.
